### PR TITLE
Fixes-"#313"

### LINKED ISSUE #313: Update links on the landing page
The following links need to be updated on the landing page: 

- [ ] "Learn about security best practices" should link to `/documentation/develop/build-a-secure-extension/`
- [ ] "learn about our policies" should link to `/documentation/publish/add-on-policies/`

### PR DESCRIPTION
**Pull request by mahimas28**
Fixes #313

### CODE REVIEW

File: _includes/home/build.md [hunk 1]
```diff
@@ -130,14 +130,14 @@ Mozilla takes the safety and privacy of its users very seriously. Be sure to rev
 <div class="grid-container grid-x grid-padding-x align-center">
 <div class="cell small-12 medium-6" markdown="1">
 
-<img src="{% asset "Learn-About-Security-Best-Practices.svg" @path @optim %}" class="image-block-first" alt="Learn about security best practices" title="Learn about security best practices">
-[Learn about security best practices](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities)
+<img src="{% asset "Learn-About-Security-Best-Practices.svg" @path @optim %}" class="image-block-first" alt="Learn about security best practices" title="Learn about security best practices" >
```
Comment (championshuttler) [inline _includes/home/build.md]: Remove the extra space from here please.

File: _includes/home/build.md [hunk 2]
```diff
@@ -130,14 +130,14 @@ Mozilla takes the safety and privacy of its users very seriously. Be sure to rev
...
+[Learn about security best practices](/documentation/develop/build-a-secure-extension.md)
```
Comment (championshuttler) [inline _includes/home/build.md]: Please remove the .md and replace it with `/`

File: _includes/home/build.md [hunk 3]
```diff
@@ -130,14 +130,14 @@ Mozilla takes the safety and privacy of its users very seriously. Be sure to rev
...
 
 </div>
 <div class="cell small-12 medium-6" markdown="1">
 
 <img src="{% asset "Learn-About-Our-Policies.svg" @path @optim %}" class="image-block-first" alt="Learn about our policies" title="Learn about our policies">
-[Learn about our policies](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities)
+[Learn about our policies](/documentation/publish/add-on-policies.md)
```
Comment (championshuttler) [inline _includes/home/build.md]: Same here.

**Comment by caitmuenster** [comment]
@mahimas28 Woohoo! I see that @championshuttler merged your PR, but unfortunately the links go to a 404. :( 

But this should be a really easy fix if you want to take one more pass at it! Replace the `.md` with a /`/ on lines 134 and 140 and those links should go live!

**Comment by championshuttler** [comment]
Sorry my bad @caitmuenster :/

**Pull request merged**